src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.html

```diff
@@ -18,7 +18,7 @@
         </div>
       </div>
 
-      <div class="mr-8" *ngIf="chartType === 'bar' || chartType === 'line'" [ngClass]="{'hide': hideElements}">
+      <div class="mr-8" *ngIf="chartType === 'bar' || chartType === 'line' && !bigNumberCharts.length" [ngClass]="{'hide': hideElements}">
```

Comment:
The condition should be restricted to only bar and line. Because even line and bar can have big number at the top

Reply:
fixed

Reply:
&& !bigNumberCharts.length - isn't removed yet